markdown/3.x/en/ff-record-list.md

```diff
@@ -128,6 +128,9 @@ Note: When you manually load a page, e.g. from an URL with the 3rd page set in i
 
 We recommend using higher values for `[infinite-debounce-delay]` and `infinite-scroll-margin` like:
 
+**Note:**
```

Comment:
Same remarks as in `1.x` file.

Reply:
changed

Reply:
Please, unify notes so they look the same in 3.x and 1.x - now they differ. I guess 1.x is the correct one.

Reply:
done